js/samples/04.ai.a.teamsChefBot/.vscode/launch.json

```diff
@@ -90,6 +90,18 @@
                 "order": 2
             },
             "stopAll": true
+        },
```

Comment:
It seems like this code belongs in the other PR. If so could you please remove this from this PR. Thanks